blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+
+## Creating a Windows VM using Microsoft Dev Box
+
+The second Windows VM option I tried was [Microsoft Dev Box](https://azure.microsoft.com/products/dev-box/). Microsoft Dev Box is a [Microsoft Azure](https://azure.microsoft.com) product that allows developers to spin up a Windows VM from most modern devices including MacOS devices.
+
+Microsoft Dev Box supports:
+
+1. Cloud-based virtual desktop infrastructure.
+1. Project-based dev box configurations which allow for code repositories and tools for your current task to be preinstalled.
+1. Region-specific dev boxes to help ensure that dev team members have a high-fidelity experience anywhere in the world.
+1. Secured cloud workstations via Microsoft Intune.
+
+Before you get started creating a VM with Microsoft Dev Box, there are a couple troubleshooting callouts I want to mention:
+
+1. You will need an Azure subscription that is not the ["free trial"](https://azure.microsoft.com/free/) subscription. You must upgrade to the [“Pay As You Go”](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go/) subscription to use Microsoft Dev Box, but you can use the $200 of free credits that you gained via the "free trial" subscription as initial payment within the “Pay As You Go” plan. If you try to create a Dev Box Definition with the "free trial" Azure subscription, you will received a "Quota has been reached" error.
+1. You will also need a work or school MSA (Microsoft Account). Personal or guest MSA’s cannot be used with the Dev Box portal at this time.
+1. Your machine will need to be managed with [Microsoft Intune](https://www.microsoft.com/security/business/Microsoft-Intune).
+
+I followed Microsoft's documentation to create a dev box using Microsoft Dev Box. I found the documentation to be straightforward to follow; there are also several tutorial videos online from developers, if you prefer to see someone walk through the steps live. You'll need to complete two steps of documentation to get your dev box up and running: [Setting Up Your Dev Box Service](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) and [Creating a Dev Box](https://learn.microsoft.com/azure/dev-box/quickstart-create-dev-box). These steps should take about an hour to complete.
+
+Once your dev box is running, you will be able to use the virtual machine as if you are on a Windows computer.
+
+From here, you can follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Dev box will not have access to your local files, so you will need to redownload any source code that you need to get started developing. When you are finished with your development session and have shut down the virtual machine in the Dev Box portal, your machine's state will be saved for next time.
```

Comment:
There are many permutations of "dev" and "box" throughout this section. Is it: "Microsoft DevBox", "DevBox", "Dev Box", or "dev box"? Does the official documentation have consistency?

Reply:
I believe the product name is "Microsoft Dev Box", and then when referring to an instance of the Windows VM they use "dev box"

Reply:
Took a pass and looks like all my mentions fit those naming conventions. Let me know if you see any deviations.

Reply:
Perfect, thank you for checking and cleaning